.github/workflows/sgx.yml

```diff
@@ -0,0 +1,56 @@
+name: sgx
+
+on:
+  push:
+    branches: [ 'sgx**' ]
+  pull_request:
+    branches: [ 'sgx**' ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-18.04
+    container: 
+      image: baiduxlab/sgx-rust:1804-1.1.1
```

Comment:
Can we use this image `teaclave/teaclave-build-ubuntu-1804-sgx-2.9:0.1.0`? Thanks.

Reply:
No problem